05/app.js

```diff
@@ -0,0 +1,64 @@
+function Student( firstName , secondName ) {
+
+    this.firstName = firstName,
+    this.secondName = secondName,
+
+    this.grades = {};
+}
+
+
+Student.prototype.addGrade = function(subject, grades) {
+
+
+      if(typeof this.grades[subject] === 'undefined') {
+          this.grades[subject] = [];
+          this.grades[subject].push(grades);
+
+      } else {
+          this.grades[subject].push(grades);
+      }
+}
+
+Student.prototype.getAverageGrade = function(subject) {
+
+    if( !( typeof this.grades[subject] === 'undefined' ) ) {
```

Comment:
Może czytelniej by było: `if(typeof this.grades[subject] !== 'undefined')`

Reply:
Dla mnie ten zapis jest czytelny, ale oczywiście wezmę do serca uwagę i zacznę stosować :).